packages/runtime/test/dataViews/RelationshipTemplateDVO.test.ts

```diff
@@ -142,12 +143,19 @@ describe("RelationshipTemplateDVO", () => {
                 ]
             }
         ];
-        templatorTemplate = (await createTemplate(templator.transport, templateContent)) as RelationshipTemplateDTO & { content: RelationshipTemplateContentJSON };
+        templatorTemplate = (
+            await templator.transport.relationshipTemplates.createOwnRelationshipTemplate({
+                maxNumberOfAllocations: 1,
+                expiresAt: DateTime.utc().plus({ minutes: 10 }).toString(),
+                content: templateContent,
+                forIdentity: requestor.address
+            })
+        ).value;
         templateId = templatorTemplate.id;
     });
 
     test("TemplateDVO for templator", async () => {
-        const dto = templatorTemplate;
+        const dto = templatorTemplate as RelationshipTemplateDTO & { content: RelationshipTemplateContentJSON };
```

Comment:
I don't see a reason for this change.

Reply:
Without it, I get "Property 'title' does not exist on type 'RelationshipTemplateContentDerivation'.
  Property 'title' does not exist on type 'ArbitraryRelationshipTemplateContentJSON'." in line 167

Reply:
you can re-add it above.

Reply:
(where you removed it :) )